### ISSUE TITLE
Please add these trackers

### ISSUE THREAD

**Issue by jakecharlie** [StevenBlack/hosts#2241]
From LG WebOS SmartTV:
aic-ngfts.lge.com
Blocked by notracking: [github.com/notracking/hosts-blocklists](https://github.com/notracking/hosts-blocklists)
and 1hosts: https://o0.pages.dev/Pro/adblock.txt

From Google:
userlocation.googleapis.com
Blocked by 1hosts: https://o0.pages.dev/Pro/adblock.txt

From Shopee:
log-collector.shopee.com.br
Blocked by 1hosts: https://o0.pages.dev/Pro/adblock.txt
Blocked by adguard dns too.

From whatsapp:
dit.whatsapp.net
Blocked by 1hosts: https://o0.pages.dev/Pro/adblock.txt
Blocked by adguard dns too.
you are blocking "dit.whatsapp.net.iberostar.com" that is not "dit.whatsapp.net"

**Comment by icedterminal** [comment]
`userlocation.googleapis.com` is highly unlikely to be added. It has been asked before in the past and denied. Blocking Google's APIs by default is bad in practice. They are *widely* used and blocking any one of them will result in hundreds if not thousands of complaints because websites/services are broken. Just like how the PiHole community manages a list of domains that simply cannot be blocked, even if they include a tiny amount of tracking. https://discourse.pi-hole.net/t/commonly-whitelisted-domains/212

This domain is largely used on Android to facilitate user location for apps. If blocked, the API cannot gather location and provide this information. This means apps may not work if location is required to function.

**Comment by icedterminal** [comment]
@StevenBlack pinging you for clarifying these additional domains.

aic-ngfts.lge.com is an ad delivery domain for LG webOS TVs. Luckily I have an LG TV. I have verified that blocking this domain does not interfere with the normal functions of webOS. 

log-collector.shopee.com.br is used in the services apps. I don't see it in use on the website. It appears to collect logs at intervals. Used an AVD for this and left it running for hours with the app open, idle. What these logs contains I do not know 100%. But it's likely it includes some form of tracking unique information.

dit.whatsapp.net is interesting.

> DIT will eventually allow us to obtain usage, reliability, and performance data about our service in a de-identified, privacy-protective way—effectively making WhatsApp even more private. For example, we would be able to understand things like how many people have outdated operating system software or which version of WhatsApp they are running without knowing who those people are. We could also understand if messages have been sent successfully without knowing who sent them. 

https://engineering.fb.com/2021/04/16/production-engineering/dit/

Given how WhatsApp is owned by Facebook, you may be doubtful of these statements. I personally do not use WhatsApp so I cannot verify whether or not this domain is critical.

**Comment by jakecharlie** [comment]
@icedterminal Hello, Thank you for your reply.

**dit.whatsapp.net** is a whatsapp telemetry domain. As whatsapp is closed source, it will never be possible to guarantee that Facebook is speaking the truth in that post you mentioned.I use whatsapp every day as my default messaging application and for several months this domain has been blocked and without any crash or problems sending or receiving messages. AdguardDNS itself blocks it.

**aic-ngfts.lge.com** Could you do me a favor? Could you prove to me that this domain is indeed LG ads. I have LG, I know it's about ads, but I have another topic open on AdguardDNSFilter's Github and I'm having a hard time convincing them that it's from advertisements, because they don't have an LG webos TV like us. Maybe a printout of packet monitoring, something like that.

Thank you very much

**Comment by icedterminal** [comment]
> **aic-ngfts.lge.com** Could you do me a favor? Could you prove to me that this domain is indeed LG ads. I have LG, I know it's about ads, but I have another topic open on AdguardDNSFilter's Github and I'm having a hard time convincing them that it's from advertisements, because they don't have an LG webos TV like us. Maybe a printout of packet monitoring, something like that.

I'm not sure why they would give you a hard time. I searched the web and found this: https://discourse.pi-hole.net/t/commonly-blacklisted-domains/305/24 - Then added the variation you have suggested. This domain is just a newer, alternative of `ngfts.lge.com`. The "tradeoffs" mentioned in the reply are not something I experience. Perhaps that was a 2017 issue, but not now. At least in my case. It literally only made one ad section go away. As the rest were already blocked. Have you not been able to prove this already? All it takes is a photo of the UI with your adblocking enabled vs disabled.

> dit.whatsapp.net is a whatsapp telemetry domain. As whatsapp is closed source, it will never be possible to guarantee that Facebook is speaking the truth in that post you mentioned.I use whatsapp every day as my default messaging application and for several months this domain has been blocked and without any crash or problems sending or receiving messages. AdguardDNS itself blocks it.

Understood.

**Comment by jakecharlie** [comment]
@icedterminal 

Unfortunately I don't have the technical knowledge you have. If you can help me there with this and other domains, just as you are helping here, it will be very welcome. thank you very much

https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1226

**Comment by hagezi** [comment]
`aic-ngfts.lge.com` - Breaks LG Content App Cover Art, see https://github.com/hagezi/dns-blocklists/issues/309#issuecomment-1422014156

**Comment by icedterminal** [comment]
Thanks for your additional feedback @hagezi .

I currently do not experience this. I may sooner or later. In this case it may be best to decline adding it.
